python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -120,6 +122,8 @@ def __init__(self, op: PhysicalOperator, inqueues: List[Deque[MaybeRefBundle]]):
         self.progress_bar = None
         self.num_completed_tasks = 0
         self.inputs_done_called = False
+        # Tracks whether each input op will no longer have input added to it.
```

Comment:
```suggestion
        # Tracks whether `input_done` is called for each input op. 
```
Nit, strictly speaking, "no longer have input added to it" happens before "`input_done` is called". So this is will be more accurate.